### ISSUE TITLE
PHP - 'Vitess\Proto\Vtgate\ExecuteResponse' does not have a method 'deserialize'

### ISSUE THREAD

**Issue by Rastusik** [youtube/vitess#1879]
Hi, I think there is a problem with the new Proto PHP class - ExecuteResponse ...

@mdhheydari opened an issue with the PDO driver (https://github.com/pixelfederation/vitess-php-pdo/issues/6), but the problem seems to be in the Vitess client:

```bash
PHP Warning: call_user_func() expects parameter 1 to be a valid callback, class 'Vitess\Proto\Vtgate\ExecuteResponse' does not have a method 'deserialize' in vendor/grpc/grpc/src/php/lib/Grpc/AbstractCall.php on line 115
PHP Fatal error: Uncaught Error: Call to a member function getError() on null in vendor/vitess/php-client/php/src/Vitess/ProtoUtils.php:35
Stack trace:
#0 vendor/vitess/php-client/php/src/Vitess/VTGateConn.php(53): Vitess\ProtoUtils::checkError(NULL)
#1 vendor/pixelfederation/vitess-php-pdo/src/VitessPdo/PDO/Vitess/Vitess.php(207): Vitess\VTGateConn->execute(Object(Vitess\Context), 'select * from u...', Array, 2)
#2 vendor/pixelfederation/vitess-php-pdo/src/VitessPdo/PDO/PDOStatement.php(152): VitessPdo\PDO\Vitess\Vitess->executeRead(Object(VitessPdo\PDO\QueryAnalyzer\PartiallyParsedQuery), Array)
#3 vendor/pixelfederation/vitess-php-pdo/src/VitessPdo/PDO.php(190): VitessPdo\PDO\PDOStatement->execute()
#4 test2.php(9): VitessPdo\PDO->query('select * from u...')
#5 {main}
thrown in vendor/vitess/php-client/php/src/Vitess/ProtoUtils.php on line 35
```

**Comment by mdhheydari** [comment]
I finally figured it out, It happens because composer grabs 'datto/Protobuf-PHP' instead of 'stanley-cheung/Protobuf-PHP'.

**Comment by Rastusik** [comment]
@mdhheydari are you sure? I did a composer update in my root vitess project, which always had a correct link to the stanley-cheung/Protobuf-PHP library and I'm getting the same error.

**Comment by Rastusik** [comment]
I know what happened ... the fork in stanley-cheung/protobuf-php changed the package name, so it doesn't override dattoo/protobuf-php anymore.

**Comment by enisoc** [comment]
This should be fixed by #1880 and https://github.com/youtube/vitess/pull/1884.
